Ivan_Shytskyi/Lesson_5/Lesson_5.Homework/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+
+namespace Lesson_5.Homework
+{
+    class Program
+    {
+        static int GCD (int x, int y)
+        {
+            while (y != 0)
+            {
+                if (x > y)
+                {
+                    int c = x % y;
+                    x = y;
+                    y = c;
+                }
+                else
```

Comment:
this else statement code is redundant, after first iteration x and y will be swaped